### DISCUSSION TITLE
[11 gennaio 2022 - B] Esercizio 3

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#201] [Esami]
**Esercizio 3 (3 punti)**

Usando gli assiomi dell’algebra di Boole, verificare la seguente identità:

![image](https://user-images.githubusercontent.com/69389461/211828537-79cefbd8-f4bf-4564-9a7c-7b73d34f2b55.png)

**Comment by Elia-Belli** [comment]
$(ab+c)+(\overline {a+bc}) + ac + \bar a \bar c$
$ab+c+(\bar a (\bar b + \bar c)) + ac + \bar a \bar c$
$ab + c + \bar a \bar b + \bar a \bar c$
$ab + \bar a \bar b + (c + \bar c)(c + \bar a)$
$c + \bar a + ab$
$c + (\bar a + a)(\bar a +b)$
$c + \bar a + b$

**Comment by fraxb04** [comment]
@Elia-Belli 
corretto
